<?xml version="1.0"?>
<launch>
    <include file="$(find assignment_2_2022)/launch/sim_w1.launch" />
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "1.0" />
    <param name="pub_freq" value= "10" />
    <node pkg="assignment_2_2022" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="assignment_2_2022" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="assignment_2_2022" type="nodeAclient.py" name="nodeAclient" output="screen"/>
    <node pkg="assignment_2_2022" type="nodeApub.py" name="nodeApub" />
    <node pkg="assignment_2_2022" type="nodeB.py" name="nodeB" />
    <node pkg="assignment_2_2022" type="nodeC.py" name="nodeC" />
    <node pkg="assignment_2_2022" type="bug_as.py" name="bug_action_service" output="screen" />
</launch>
